discord/scheduled_event.py

```diff
@@ -51,10 +53,225 @@
 # fmt: off
 __all__ = (
     "ScheduledEvent",
+    "ScheduledEventRecurrence",
+    "ScheduledEventExceptionCount"
 )
 # fmt: on
 
 
+class ScheduledEventExceptionCount:
+    """Represents the exception counts in a Scheduled Event.
+    
+    .. versionadded:: 2.4
+
+    .. container:: operations
+
+        .. describe:: x == y
+
+            Checks if two Exception Counts are equal.
+    """
+
+    def __init__(self, data: GuildScheduledEventExceptionCountsPayload) -> None:
+        self.count: int = int(data.get('guild_scheduled_event_count'))
+
+        self._exception_snowflakes: Dict[Union[str, int], int] = data.get('guild_scheduled_event_exception_counts')
+
+    def __eq__(self, other: object) -> bool:
+        if isinstance(other, self.__class__):
+            return self.exception_ids == other.exception_ids
+        return NotImplemented
+
+    @property
+    def exception_ids(self) -> List[int]:
+        """List[:class:`int`]: A list containing all the exception event IDs"""
+        return [int(id) for id in self._exception_snowflakes.keys()]
+    
+    @property
+    def exceptions(self) -> Dict[int, int]:
+        """Dict[:class:`int`, :class:`int`]: A dictionary containing all the
+        event IDs as keys and their respective exception counts as value.
+        """
+
+        return {int(snowflake): count for snowflake, count in self._exception_snowflakes.items()}
+
+
+class ScheduledEventRecurrence:
+    """Represents a Scheduled Event Recurrence
+    
+    .. versionadded:: 2.4
+
+    .. container:: operations
+
+        .. describe:: x == y
+
+            Checks if two Scheduled Event Recurrences are equal
+
+    Parameters
+    ----------
+    start: :class:`datetime.datetime`
+        When the first event of this series is started.
+    end: Optional[:class:`datetime.datetime`]
+        When the events of this series will stop. If it is `None`, it will repeat forever.
+    weekday: :class:`int`
+        An integer representing the weekday this event will repeat in. Monday is 0
+        and Sunday is 6.
+    n_weekday: Tuple[:class:`int`, :class:`int`]
+        A tuple that contain the N weekday this event will repeat in.
+
+        For example, if you want for this event to repeat the 1st Monday of the month,
+        then this param should have a value of `(1, 0)`. Where ``1`` represents the
+        'first' and ``0`` the weekday, in this case, Monday.
```

Comment:
Instead of an example easily missed here, why not do something like [this](https://discordpy.readthedocs.io/en/stable/api.html#discord.FFmpegOpusAudio.from_probe) with a few (small) examples.